server.js

```diff
@@ -0,0 +1,30 @@
+import express from "express";
+
+// const express = require('express')
+
+const app = express();
+const port = 3025;
+
+const jason = {
+	Todos: [
+		{
+			name: "Sebastian",
+			city: "Hamburg",
+		},
+		{
+			name: "Florian",
+			city: "Frankfurt",
+		},
+	],
+};
+app.get("/", (req, res) => {
+	res.send("Hello World");
+});
+
+app.get("/todos/", (request, response) => {
```

Comment:
A better naming convention would be: `"/api/todos"` for this endpoint, because it's not a "normal route"